spec/user_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Mexico.', posts_counter: 0) }
+
+  before { subject.save }
+
```

Comment:
- [ ] Kindly add some +ve test cases.

Reply:
Hello @Deepakdanger I am not understanding what you mean by +ve test cases